common/djangoapps/student/views.py

```diff
@@ -318,6 +318,30 @@ def get_course_enrollments(user, orgs_to_include, orgs_to_exclude):
             yield enrollment
 
 
+def get_course_certificates(user):
+    """
+    Returns information about a user's course certificates.
+    """
+    # Let's filter out any courses in an "org" that has been declared to be
+    # in a configuration
```

Comment:
Nit: Just to keep consistency in comments (starting with an active verb), start this with "Filter out..."